Gemfile.lock

```diff
@@ -84,16 +85,21 @@ GEM
       ruby-progressbar (~> 1.7)
       unicode-display_width (~> 1.0, >= 1.0.1)
     ruby-progressbar (1.9.0)
-    simplecov (0.11.2)
-      docile (~> 1.1.0)
-      json (~> 1.8)
+    set (1.1.0)
+    simplecov (0.17.1)
```

Comment:
bumped simplecov because the underlying json version for the previous version is not compatible with ruby 3.x